controllers/registry.go

```diff
@@ -532,6 +532,20 @@ func (r *DPAReconciler) getProviderSecret(secretName string) (corev1.Secret, err
 	r.Log.Info(fmt.Sprintf("got provider secret name: %s", secret.Name))
 	return secret, nil
 }
+func (r *DPAReconciler) getSecretNameAndKeyforBackupLocation(bslspec oadpv1alpha1.BackupLocation) (string, string) {
+	var secretName, secretKey string
+	if bslspec.CloudStorage != nil {
+		if bslspec.CloudStorage.Credential != nil {
+			secretName = bslspec.CloudStorage.Credential.Name
+			secretKey = bslspec.CloudStorage.Credential.Key
```

Comment:
NIT:  maybe return here instead of continuing down the func?

```
return bslspec.CloudStorage.Credential.Name, bslspec.CloudStorage.Credential.Key
```